src/igvfd/schemas/changelogs/mixins.md

```diff
@@ -1,5 +1,7 @@
 ## Changelog for *`mixins.json`*
 
+* Add linking to reference files for `large_scale_gene_list` and `large_scale_loci_list`. (02/14/2024)
```

Comment:
Add ReferenceFile linkTo to large_scale_gene_list and large_scale_loci_list.

Reply:
changed